tools/loader/ImageClassifier.cpp

```diff
@@ -218,6 +224,31 @@ static void printTopKPairs(const std::vector<FloatIndexPair> &topKPairs) {
   }
 }
 
+/// Checks if \p topKPairs have the index that matches the provided index,
+/// \returns the number of found mismatches.
+static int matchIndices(const std::vector<FloatIndexPair> &topKPairs,
```

Comment:
nit: Use `llvm::ArrayRef<FloatIndexPair> topKPairs` instead of const ref.

Reply:
Resolved.